crates/ethcore/src/json_tests/local.rs

```diff
@@ -0,0 +1,203 @@
+use super::HookType;
+use ethereum_types::U256;
+use ethjson::{self, blockchain::Block};
+use log::warn;
+use rlp::RlpStream;
+use std::path::Path;
+use types::transaction::{TypedTransaction, TypedTxId, UnverifiedTransaction};
+use verification::queue::kind::blocks::Unverified;
+
+pub fn json_local_block_en_de_test<H: FnMut(&str, HookType)>(
+    _test: &ethjson::test::LocalTests,
+    path: &Path,
+    json_data: &[u8],
+    start_stop_hook: &mut H,
+) -> Vec<String> {
+    let tests = ethjson::local_tests::BlockEnDeTest::load(json_data).expect(&format!(
+        "Could not parse JSON chain test data from {}",
+        path.display()
+    ));
+    let mut failed = Vec::new();
+
+    for (name, ref_block) in tests.into_iter() {
+        start_stop_hook(&name, HookType::OnStart);
+
+        let block = Unverified::from_rlp(ref_block.rlp());
+        let block = match block {
+            Ok(block) => block,
+            Err(decoder_err) => {
+                warn!(target: "json-tests", "Error decoding test block: {:?}", decoder_err);
+                failed.push(name.clone());
+                continue;
+            }
+        };
+        if !is_same_block(&ref_block, &block) {
+            println!("block failed {:?}", block);
```

Comment:
`err!`?

Reply:
Test runner uses println for printing error